### ISSUE TITLE
Support multiple budgets

### ISSUE THREAD

**Issue by ohthehugemanatee** [nuc/db-to-ynab#1]
It's not clear in the README how to select the budget to use. Looking at the code, I gather it takes the first one YNAB opens - I guess that's your most recent budget?

anyway this could easily be another constant.

**Comment by nuc** [comment]
What do you mean "budget"? Isn't that the `YNAB_ACCOUNT_TITLE`?

**Comment by ohthehugemanatee** [comment]
No, not the same as account. YNAB lets you have more than one Budget. Each budget has accounts. See the dropdown menu in the top left of the ynab interface.

For example, I have a budget for my personal finances, and one for my business. You can create a budget by hand, or you get a new one every time you make a "fresh start". long time users of YNAB probably have several budgets to choose from.
